src/test/codegen/issue-86106.rs

```diff
@@ -0,0 +1,56 @@
+// min-llvm-version: 15.0
+// compile-flags: -C opt-level=3 -C target-cpu=native
```

Comment:
I don't think we need `target-cpu=native`.

Reply:
Thank you! Fixed with other comments via https://github.com/rust-lang/rust/pull/106036/commits/3b16aea1237617578451d947a87f057fcc1250b3.